src/Raven.Server/Config/Categories/IndexingConfiguration.cs

```diff
@@ -334,6 +335,12 @@ public virtual bool RunInMemory
         [ConfigurationEntry("Indexing.Throttling.TimeIntervalInMs", ConfigurationEntryScope.ServerWideOrPerDatabaseOrPerIndex)]
         public TimeSetting? ThrottlingTimeInterval { get; protected set; }
 
+        [Description("Search engine for new AutoIndexes")]
+        [DefaultValue(Documents.Indexes.SearchEngine.Corax)]
+        [IndexUpdateType(IndexUpdateType.Reset)]
+        [ConfigurationEntry("Indexing.Auto.SearchEngine", ConfigurationEntryScope.ServerWideOrPerDatabaseOrPerIndex)]
```

Comment:
Just a note `ServerWideOrPerDatabaseOrPerIndex` probably not relevant for auto indexes, we'll actually only be able to set it on `ServerWideOrPerDatabase`, no?

Reply:
Yes, I will change this.